transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/ActionModeCallbackInterceptor.java

```diff
@@ -0,0 +1,28 @@
+package com.hyperwallet.android.ui.transfermethod.view.widget;
+
+import android.view.ActionMode;
+import android.view.Menu;
+import android.view.MenuItem;
+
+public class ActionModeCallbackInterceptor implements ActionMode.Callback {
+
+    @Override
+    public boolean onCreateActionMode(ActionMode mode, Menu menu) {
+        return false;
```

Comment:
We should still allow Paste. Only cut and copy is disabled.

Reply:
This code also allows paste when long press. However, I customized now with the paste options.